src/components/DateRangePicker.js

```diff
@@ -367,13 +381,19 @@ class DateRangePicker extends React.Component {
                     />
                   </div>
                   <div style={styles.calendarWeek}>
-                    {['S', 'M', 'T', 'W', 'T', 'F', 'S'].map((day, i) => {
-                      return (
-                        <div key={day + i} style={styles.calendarWeekDay}>
-                          {day}
-                        </div>
-                      );
-                    })}
+                    {[{ label: 'S', value: 'Sunday' },
+                      { label: 'M', value: 'Monday' },
+                      { label: 'T', value: 'Tuesday' },
+                      { label: 'W', value: 'Wednesday' },
+                      { label: 'T', value: 'Thursday' },
+                      { label: 'F', value: 'Friday' },
+                      { label: 'S', value: 'Saturday' }].map((day) => {
```

Comment:
wondering why you decided to create objects here - what is the `value` key giving you - from what i see you're using it as the key prop, which seems like overkill?